pkg/agent/config.go

```diff
@@ -165,4 +165,14 @@ type Config struct {
 	PCAFilters string `env:"PCA_FILTER"`
 	// PCAServerPort is the port PCA Server starts at, when ENABLE_PCA variable is set to true.
 	PCAServerPort int `env:"PCA_SERVER_PORT" envDefault:"9990"`
+	// MetricsPromoEnable enables prometheus server to collect ebpf agent metrics, default is false.
+	MetricsPromoEnable bool `env:"METRICS_PROMO_ENABLE" envDefault:"false"`
+	// MetricsPromoServerAddress is the address of the prometheus server that collects ebpf agent metrics.
+	MetricsPromoServerAddress string `env:"METRICS_PROMO_SERVER_ADDRESS"`
+	// MetricsPromoPort is the port of the prometheus server that collects ebpf agent metrics.
+	MetricsPromoPort int `env:"METRICS_PROMO_PORT" envDefault:"9090"`
```

Comment:
I'm curious about this "Promo" in the name, I guess it stands for "prometheus" but it's the first time I see it abbreviated like that ... anyway, I'm not sure we need to tell about Prometheus in the variable names, as it is the de-facto standard in k8s anyway. Can we just say "MetricsEnable" (or "EnableMetrics" to be more consistent with the other EnableSomething in the settings?), MetricsServerAddress, etc.